MAUI/Cartesian-Charts/DataLabels.md

```diff
@@ -146,3 +146,100 @@ chart.Series.Add(series);
 {% endtabs %}
 
 ![Applying series interior for data label in MAUI chart](DataLabel_images/maui_chart_data_label_with_series_brush.jpg)
+
+## Formatting label context
+
+You can customize the content of the label using [LabelContext]() property. Following are the two options that are supported now,
+
+* [Percentage]() - This will show the percentage value of corresponding data point Y value
```

Comment:
Need to update the most commonly used series type with Label Context - update as Bar, Column, Stacked bar & Stacked column